js/foam/apps/calc/CalcView.js

```diff
@@ -281,18 +284,18 @@ CLASS({
       justify-content: center;
       display: flex;
       align-items: center;
-      background-color: #4b4b4b;
+      background-color: #333333;
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation

Reply:
Thank you!